### PR TITLE
feat(config): add support for MP21Z

### PR DESCRIPTION
**Pull request by highaltidude**

### CODE REVIEW

File: packages/config/config/devices/0x0312/mp21z.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      "description": "Count Down Configuration",
+      "valueSize": "2",
+      "minValue": "0",
+      "maxValue": "32767",
+      "defaultValue": "0",
+      "readOnly": false,
+			"writeOnly": false,
+      "allowManualEntry": true
+    },
+  }
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21z.json]: comma missing

File: packages/config/config/devices/0x0312/mp21z.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      },
+      {
+        "label": "LED off",
+        "value": 2
+      }
+      ]
+    },
+    "2":{
+      "label": "Count Down Configuration",
+      "description": "Count Down Configuration",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21z.json]: If the description does not add value, better remove it.

Comment (AlCalzone) [reply packages/config/config/devices/0x0312/mp21z.json]: Or maybe there's a better explanation for this param?

File: packages/config/config/devices/0x0312/mp21z.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      "valueSize": "2",
+      "minValue": "0",
+      "maxValue": "32767",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21z.json]: Is there a unit for these values?

File: packages/config/config/devices/0x0312/mp21z.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      "defaultValue": "0",
+      "readOnly": false,
+			"writeOnly": false,
+      "allowManualEntry": true
+    },
+  }
+  "metadata": {
+		"inclusion": "Refer to your primary controller instructions to process the inclusion \n exclusion setup procedure. When prompted by your primary controller, \n click the Up or Down button three times in one second. Include MP21Z to \n from a Z-Wave Gateway with supporting Security.",
+		"exclusion": "Refer to your primary controller instructions to process the inclusion \n exclusion setup procedure.",
+		"reset": "Power off MP21Z keep holding the PROG button Power on,Press and keep \n holding the PROG button for no less than 10 seconds. Please use this procedure \n only when the network primary controller is missing or otherwise inoperable."
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21z.json]: These could use some cleaning (remove the unnecessary \n, add missing space after ",")

File: packages/config/config/devices/0x0312/mp21z.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        "value": 0
+      },
+      {
+        "label": "Outlet on, led off",
+        "value": 1
+      },
+      {
+        "label": "LED off",
+        "value": 2
+      }
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21z.json]: Please make this consistent (same casing for on/off and LED everywhere)

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by highaltidude** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0312/mp21z.json (hunk 3 above)
Comment (highaltidude) [reply packages/config/config/devices/0x0312/mp21z.json]: added unit

**Review by highaltidude — COMMENTED** [review]
Edits requested should be resolved.

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — DISMISSED** [review]

**Comment by AlCalzone** [comment]
Thanks, will merge once tests are through

**Comment by AlCalzone** [comment]
well... looks like you found a bug in my checks :D
Fixed both the checks and the file.

**Comment by Papester** [comment]
It seems the status indicator config option is reversed as described at https://devices.zwave-js.io/?jumpTo=0x0312:0xff00:0xff0c:1.0#

Options:
    0: LED on when load off, LED off when load on (actually is LED on when load on, LED off when load off)
    1: LED on when load on, LED off when load off (actually is  LED on when load off, LED off when load on)
    2: LED always off

Using Z-Wave JS to MQTT Current version: 0.9.1 and Z-Wave JS Current version: 0.1.16

**Comment by AlCalzone** [comment]
Can you open a PR with the change please? The device browser should point you to the correct file.

**Pull request merged**